tests/functional/reporting-non-auth.js

```diff
@@ -286,30 +268,23 @@ define(
           .end();
       },
 
-      "details param adds info to description": function() {
```

Comment:
in the review comments, you asked me to move adding the details query string to the text area at form submit time. in a different pr, you also explained that we shouldn't write tests that create an actual issue in order to avoid getting labeled as spammers by github. it seems like if I test the form submit (clicking submit buttons), this would  be creating a new github issue? so I believed there was no practical way to test the newly requested behavior(??)